model/src/main/kotlin/OrtResult.kt

```diff
@@ -412,13 +412,19 @@ data class OrtResult(
     fun getPackage(id: Identifier): CuratedPackage? = packages[id]?.curatedPackage
 
     @JsonIgnore
-    fun getPackages(): Set<CuratedPackage> = analyzer?.result?.packages.orEmpty()
+    fun getPackages(omitExcluded: Boolean = false): Set<CuratedPackage> =
```

Comment:
Commit message: Let's not make a mystery about *what* param gets added and change the title to

    OrtResult: Add a param to getProjects() and getPackages() to omit excludes

Reply:
I remember you asked me to pay more attention to the number of chars limit for the commit message title. Thus you have two requests to me which are conflicting, can you please clarify?

Reply:
There's a hard limit of 75 chars (to make `git log` read nice in a classic terminal), but this title has only 74 chars. GitHub promotes its own limit of 50 chars for the title, but I regard that as a soft limit (i.e. recommendation) as 50 chars really is quite short, and to me not GitHub but Git itself is the definite source for Git best practices.

Reply:
Done.